docs/admin/deployment.md

```diff
@@ -1099,51 +1102,46 @@ fabric_ifaces:
 
 ### Agent Startup
 
-DAOS Agent is a standalone application to be run on each compute node.
-It can be configured to use secure communications (default) or can be allowed
-to communicate with the control plane over unencrypted channels. The following
-example shows `daos_agent` being configured to operate in insecure mode due to
-incomplete integration of certificate support as of the 0.6 release and
-configured to use a non-default agent configuration file.
+The DAOS Agent is a standalone application to be run on each client node.
+By default, the DAOS Agent will be run as a systemd service.
+The DAOS Agent unit file is installed in the correct location
+(`/usr/lib/systemd/system/daos_agent.service`) during RPM installation.
 
-To start the DAOS Agent from the command line, run:
+After the RPM installation, and after the Agent configuration file has
+been created, the following commands will enable the DAOS Agent to be
+started at the next reboot, will start it immediately,
+and will check the status of the Agent after being started:
 
 ```bash
-$ daos_agent -i -o <'path to agent configuration file/daos_agent.yml'> &
+$ sudo systemctl enable daos_agent.service
+$ sudo systemctl start  daos_agent.service
+$ sudo systemctl status daos_agent.service
 ```
 
-Alternatively, the DAOS Agent can be started as a systemd service. The DAOS Agent
-unit file is installed in the correct location when installing from RPMs.
-If you want to run the DAOS Agent without certificates (not recommended in production
-deployments), you need to add the `-i` option to the systemd `ExecStart` invocation
-(see below).
-
-If you wish to use systemd with a development build, you must copy the service
-file from `utils/systemd` to `/usr/lib/systemd/system`. Once the file is copied
-modify the ExecStart line to point to your in tree `daos_agent` binary.
-
-`ExecStart=/usr/bin/daos_agent -i -o <'path to agent configuration file/daos_agent.yml'>`
-
-Once the service file is installed, you can start `daos_agent`
-with the following commands:
+If the DAOS Agent fails to start, check the systemd logs for errors:
 
 ```bash
-$ sudo systemctl daemon-reload
-$ sudo systemctl enable daos_agent.service
-$ sudo systemctl start daos_agent.service
+$ sudo journalctl --unit daos_agent.service
 ```
 
-To check the component status use:
+#### Starting the DAOS Agent with a non-default configuration
+
+To start the DAOS Agent from the command line, for example to run with a
+non-default configuration file, run:
 
 ```bash
-$ sudo systemctl status daos_agent.service
+$ daos_agent -i -o <'path to agent configuration file/daos_agent.yml'> &
```

Comment:
Let's remove the -i suggestion - that would start the agent in insecure mode.

Reply:
fixed